geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/io/LayerReaderFactory.scala

```diff
@@ -63,6 +63,9 @@ abstract class FilteringLayerReaderWrapper()
   def attributeStore: AttributeStore
   def layerReader: FilteringLayerReader[LayerId]
 
+  def layerIds: Array[String] =
+    attributeStore.layerIds.map { x => x.toString }.toArray
```

Comment:
Is .toString actually useful here? It will return something like `Layer(name = "asdf", zoom = 3)` which some poor soul would have to parse. Probably we can return a list of dictionaries with `name` and `zoom` keys here.